custom-connectors/AzureADRiskyUsers/apiDefinition.swagger.json

```diff
@@ -0,0 +1,239 @@
+                      },
+                      "isDeleted": {
+                        "type": "boolean",
+                        "description": "isDeleted"
+                      },
+                      "isProcessing": {
+                        "type": "boolean",
+                        "description": "isProcessing"
+                      },
+                      "riskLevel": {
+                        "type": "string",
+                        "description": "riskLevel",
+                        "title": "",
+                        "x-ms-visibility": "important"
+                      },
+                      "riskState": {
+                        "type": "string",
+                        "description": "riskState"
+                      },
+                      "riskDetail": {
+                        "type": "string",
+                        "description": "riskDetail",
+                        "title": "",
+                        "x-ms-visibility": "important"
+                      },
+                      "riskLastUpdatedDateTime": {
+                        "type": "string",
+                        "description": "riskLastUpdatedDateTime",
+                        "title": "",
+                        "x-ms-visibility": "important"
+                      },
+                      "userDisplayName": {
+                        "type": "string",
+                        "description": "userDisplayName",
+                        "title": "",
+                        "x-ms-visibility": "important"
+                      },
+                      "userPrincipalName": {
+                        "type": "string",
+                        "description": "userPrincipalName",
+                        "title": "",
+                        "x-ms-visibility": "important"
+                      }
+                    }
+                  },
+                  "description": "value"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/beta/riskyUsers/dismiss": {
+      "post": {
+        "summary": "Dismiss Risky user",
+        "description": "Dismiss user risk",
+        "operationId": "DismissRiskyUser",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "userIds": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "description": "userIds",
+                    "title": "userIds"
+                  },
+                  "description": "userIds"
+                }
+              },
+              "default": {
+                "userIds": [
+                  "{{userID}}"
+                ]
+              }
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "204",
```

Comment:
@BakkerJan The response descriptions should not be the response numbers. It doesn't add any value to the user incase of trouble shooting issues.